framework/bin/run-benchmark.js

```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/env node
-import cli from '../src/cli/bench.js';
+import {
+  cli, run,
+} from '../src/cli/bench.js';
 
 cli(process.argv,);
```

Comment:
this already runs the process. move the argument parsing into cli :)

Reply:
Oh, okay I see it now, sorry :) Can I use args array and constants that are already provided?

Reply:
of course